src/main/kotlin/me/modmuss50/mpp/platforms/discord/DiscordAPI.kt

```diff
@@ -10,7 +10,7 @@ import okhttp3.RequestBody.Companion.toRequestBody
 
 object DiscordAPI {
     @OptIn(ExperimentalSerializationApi::class)
-    val json = Json { explicitNulls = false }
+    val json = Json { explicitNulls = false; classDiscriminator = "class"; encodeDefaults = true }
```

Comment:
Question: Why is classDiscriminator = "class" needed?

Reply:
A component uses the field `type`, which is de default discriminator, causing a conflict, this was needed to fix that